app/resources/create.html.template

```diff
@@ -40,15 +40,10 @@
               {% endifequal %}
             </h1>
           </div>
-          <div class="name section">
+          <div class="identifyinfo section">
             <h2>
-              {% trans "Identifying information" %}
+            <center>{% trans "Identifying information" %}</center>
             </h2>
-
-            <div class="subsection">
-              <h3>
-                {% trans "Name" %}
-              </h3>
               <div class="hint">
```

Comment:
Align tags in the same level at the same column. e.g., The previous ```<h2>``` and this ```<div>``` are now in the same level, so unindent this ```<div>``` (and the following tags) to the same column as ```<h2>```.